multimedia/go2rtc/files/go2rtc.conf

```
@@ -0,0 +1,6 @@
+config daemon 'daemon'
+	option disable_jail         0
+	option user                 go2rtc
+	option group                go2rtc
+	option limit_nofile         ''
+    option allow_config_edit    0
```

I think positive toggle `use_jail` or similar is easier to reason about. Option values should be quoted. Note the indentation change.
```suggestion
	option use_jail             '1'
	option user                 'go2rtc'
	option group                'go2rtc'
	option limit_nofile         ''
	option allow_config_edit    '0'
```

---

Since jail improves security a bit and allows to apply capabilities.json, I consider it as a default on.
UCI then could drop zero valued bool options. That's why I prefer explicit `disabled` flags like in many other packages (e.g. disabled wg peer in network).

Running as a root without a jail may be needed in quite specific case, when you need to access some /dev, e.g. USB camera or other v4l device.

---

Fixed spaces to tabs.